lib/item.rb

```diff
@@ -0,0 +1,15 @@
+class Item
+
+  attr_reader :name, :price
+
+  def initialize(name, price)
+    @name = name
+    @price = price
+    @avaialble = true
```

Comment:
Cool, so the takeaway can remove those dishes that are no longer available.

Reply:
Currently it doesn't seem like there'a method to make items unavailable, which renders this attribute less useful

Reply:
Typo - `avaialble` -> `available`. You can use a spell checker extension on your code editor to find these.